rotkehlchen/tests/api/test_users.py

```diff
@@ -887,22 +932,26 @@ def test_user_del_premium_credentials(rotkehlchen_api_server, username):
             json=data,
         )
     with patched_get:
+        assert rotki.premium is not None
         assert rotki.premium.is_active()
 
-    # Delete premium credentials for current user
```

Comment:
why did you remove this and move it down? This should stay here, above the deletion.